### DISCUSSION TITLE
dynamic node not updating in real time

### DISCUSSION THREAD

**Discussion by jesperp** [L3MON4D3/LuaSnip#469] [Q&A] [answered]
I'm trying this popular example and it doesn't seem to update the "copied" version in real time, only after doing a `<Tab>` or `<Esc>`

https://user-images.githubusercontent.com/376921/175030858-07bc7d2a-e8e3-4996-a6da-2070137a7245.mov

Are there possible options that could affect this behaviour?

**Comment by L3MON4D3** [comment, marked as answer]
Yup, you can update on any event via `ls.setup`: https://github.com/L3MON4D3/LuaSnip#config

**Reply by jesperp** [reply]
Thanks the quick reply! One thing that tripped me up was defining this in my `init.vim` and later overwriting the config in my `all.lua`..

**Reply by L3MON4D3** [reply]
Oh, that's unfortunate :D
I'm looking into adding logging semi-short term, hopefully that will catch such issues
